src/components/ApplicationGroup/AppGroupAppFilter.tsx

```diff
@@ -101,8 +103,22 @@ export default function AppGroupAppFilter() {
         return 'Search filters'
     }
 
+    const handleFilterFocus = () => {
+        appGroupFilterRef.current.focus()
+        appGroupFilterRef.current.onMenuOpen()
```

Comment:
how about firing the click event instead of these 2?

Reply:
react select ref not giving a click event